packages/shared/src/hooks/usePostComment.ts

```diff
@@ -257,6 +266,30 @@ export const usePostComment = (
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [enableShowShareNewComment]);
 
+  const hasCommentToggleQueryParam = typeof router.query.comment === 'string';
+
+  useEffect(() => {
+    if (!hasCommentToggleQueryParam || parentComment) {
+      return;
+    }
+
+    const { comment, ...queryWithoutComment } = router.query;
+
+    // TODO WT-1313-squad-welcome-post-checklist-steps correct origin
```

Comment:
@sshanzel Would you mind having a look here

Reply:
So what's the TODO here still?

Reply:
Oh, made some changes from the code way above and forgot to remove this. Will update quickly

Reply:
Have tested locally and works as intended 🚀